<!doctype html><html lang=en dir=auto><head><meta charset=utf-8><meta http-equiv=x-ua-compatible content="IE=edge"><meta name=viewport content="width=device-width,initial-scale=1,shrink-to-fit=no"><meta name=robots content="index, follow"><title>Nemafootprinter | Tucano Profile Page</title><meta name=keywords content="Paper,Bioinformatics"><meta name=description content="NemaFootPrinter: a web based software for the identification of conserved non-coding genome sequence regions between C. elegans and C. briggsae"><meta name=author content="Rambaldi Davide, Alessandro Guffanti, Paolo Morandi, Giuseppe Cassata"><link rel=canonical href=https://tucano.github.io/papers/nemafootprinter><link crossorigin=anonymous href=/assets/css/stylesheet.css rel="preload stylesheet" as=style><script defer crossorigin=anonymous src=/assets/js/highlight.js onload=hljs.initHighlightingOnLoad()></script>
<link rel=icon href=https://tucano.github.io/img/favicon/favicon.ico><link rel=icon type=image/png sizes=16x16 href=https://tucano.github.io/img/favicon/favicon-16x16.png><link rel=icon type=image/png sizes=32x32 href=https://tucano.github.io/img/favicon/favicon-32x32.png><link rel=apple-touch-icon href=https://tucano.github.io/img/favicon/apple-touch-icon.png><link rel=mask-icon href=https://tucano.github.io/img/favicon/safari-pinned-tab.png><meta name=theme-color content="#2e2e33"><meta name=msapplication-TileColor content="#2e2e33"><noscript><style>#theme-toggle,.top-link{display:none}</style><style>@media(prefers-color-scheme:dark){:root{--theme:rgb(29, 30, 32);--entry:rgb(46, 46, 51);--primary:rgb(218, 218, 219);--secondary:rgb(155, 156, 157);--tertiary:rgb(65, 66, 68);--content:rgb(196, 196, 197);--hljs-bg:rgb(46, 46, 51);--code-bg:rgb(55, 56, 62);--border:rgb(51, 51, 51)}.list{background:var(--theme)}.list:not(.dark)::-webkit-scrollbar-track{background:0 0}.list:not(.dark)::-webkit-scrollbar-thumb{border-color:var(--theme)}}</style></noscript><meta property="og:title" content="Nemafootprinter"><meta property="og:description" content="NemaFootPrinter: a web based software for the identification of conserved non-coding genome sequence regions between C. elegans and C. briggsae"><meta property="og:type" content="article"><meta property="og:url" content="https://tucano.github.io/papers/nemafootprinter/"><meta property="og:image" content="https://tucano.github.io/papers/nemafootprinter/cover.png"><meta property="article:section" content="papers"><meta property="article:published_time" content="2005-12-01T00:00:00+00:00"><meta property="article:modified_time" content="2005-12-01T00:00:00+00:00"><meta name=twitter:card content="summary_large_image"><meta name=twitter:image content="https://tucano.github.io/papers/nemafootprinter/cover.png"><meta name=twitter:title content="Nemafootprinter"><meta name=twitter:description content="NemaFootPrinter: a web based software for the identification of conserved non-coding genome sequence regions between C. elegans and C. briggsae"><script type=application/ld+json>{"@context":"https://schema.org","@type":"BreadcrumbList","itemListElement":[{"@type":"ListItem","position":1,"name":"Papers","item":"https://tucano.github.io/papers/"},{"@type":"ListItem","position":2,"name":"Nemafootprinter","item":"https://tucano.github.io/papers/nemafootprinter/"}]}</script><script type=application/ld+json>{"@context":"https://schema.org","@type":"BlogPosting","headline":"Nemafootprinter","name":"Nemafootprinter","description":"NemaFootPrinter: a web based software for the identification of conserved non-coding genome sequence regions between C. elegans and C. briggsae","keywords":["Paper","Bioinformatics"],"articleBody":"ABSTRACT Summary Background NemaFootPrinter (Nematode Transcription Factor Scan Through Philogenetic Footprinting) is a web-based software for interactive identification of conserved, non-exonic DNA segments in the genomes of C. elegans and C. briggsae. It has been implemented according to the following project specifications:a) Automated identification of orthologous gene pairs. b) Interactive selection of the boundaries of the genes to be compared. c) Pairwise sequence comparison with a range of different methods. d) Identification of putative transcription factor binding sites on conserved, non-exonic DNA segments.\nResults Starting from a C. elegans or C. briggsae gene name or identifier, the software identifies the putative ortholog (if any), based on information derived from public nematode genome annotation databases. The investigator can then retrieve the genome DNA sequences of the two orthologous genes; visualize graphically the genes’ intron/exon structure and the surrounding DNA regions; select, through an interactive graphical user interface, subsequences of the two gene regions. Using a bioinformatics toolbox (Blast2seq, Dotmatcher, Ssearch and connection to the rVista database) the investigator is able at the end of the procedure to identify and analyze significant sequences similarities, detecting the presence of transcription factor binding sites corresponding to the conserved segments. The software automatically masks exons.\nDiscussion This software is intended as a practical and intuitive tool for the researchers interested in the identification of non-exonic conserved sequence segments between C. elegans and C. briggsae. These sequences may contain regulatory transcriptional elements since they are conserved between two related, but rapidly evolving genomes. This software also highlights the power of genome annotation databases when they are conceived as an open resource and the possibilities offered by seamless integration of different web services via the http protocol.\nPUBMED NemaFootPrinter: a web based software for the identification of conserved non-coding genome sequence regions between C. elegans and C. briggsae\nBMC Bioinformatics. 2005 Dec 1;6 Suppl 4(Suppl 4):S22. doi: 10.1186/1471-2105-6-S4-S22.\nPDF NemaFootPrinter.pdf\n","wordCount":"313","inLanguage":"en","image":"https://tucano.github.io/papers/nemafootprinter/cover.png","datePublished":"2005-12-01T00:00:00Z","dateModified":"2005-12-01T00:00:00Z","author":[{"@type":"Person","name":"Rambaldi Davide"},{"@type":"Person","name":"Alessandro Guffanti"},{"@type":"Person","name":"Paolo Morandi"},{"@type":"Person","name":"Giuseppe Cassata"}],"mainEntityOfPage":{"@type":"WebPage","@id":"https://tucano.github.io/papers/nemafootprinter/"},"publisher":{"@type":"Organization","name":"Tucano Profile Page","logo":{"@type":"ImageObject","url":"https://tucano.github.io/img/favicon/favicon.ico"}}}</script></head><body id=top><script>localStorage.getItem("pref-theme")==="dark"?document.body.classList.add("dark"):localStorage.getItem("pref-theme")==="light"?document.body.classList.remove("dark"):window.matchMedia("(prefers-color-scheme: dark)").matches&&document.body.classList.add("dark")</script><header class=header><nav class=nav><div class=logo><a href=https://tucano.github.io/ accesskey=h title="Tucano (Alt + H)"><img src=https://tucano.github.io/img/logoTitle.png alt aria-label=logo height=40>Tucano</a><div class=logo-switches><button id=theme-toggle accesskey=t title="(Alt + T)"><svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24" fill="none" stroke="currentcolor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"><path d="M21 12.79A9 9 0 1111.21 3 7 7 0 0021 12.79z"/></svg><svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24" fill="none" stroke="currentcolor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"><circle cx="12" cy="12" r="5"/><line x1="12" y1="1" x2="12" y2="3"/><line x1="12" y1="21" x2="12" y2="23"/><line x1="4.22" y1="4.22" x2="5.64" y2="5.64"/><line x1="18.36" y1="18.36" x2="19.78" y2="19.78"/><line x1="1" y1="12" x2="3" y2="12"/><line x1="21" y1="12" x2="23" y2="12"/><line x1="4.22" y1="19.78" x2="5.64" y2="18.36"/><line x1="18.36" y1="5.64" x2="19.78" y2="4.22"/></svg></button></div></div><ul id=menu><li><a href=https://tucano.github.io/docs/ title=docs><span>docs</span></a></li><li><a href=https://tucano.github.io/projects/ title=projects><span>projects</span></a></li><li><a href=https://tucano.github.io/papers/ title=papers><span>papers</span></a></li><li><a href=https://tucano.github.io/archives/ title=archives><span>archives</span></a></li><li><a href=https://tucano.github.io/tags/ title=tags><span>tags</span></a></li></ul></nav></header><main class=main><article class=post-single><header class=post-header><div class=breadcrumbs><a href=https://tucano.github.io/>Home</a>&nbsp;»&nbsp;<a href=https://tucano.github.io/papers/>Papers</a></div><h1 class=post-title>Nemafootprinter</h1><div class=post-description>NemaFootPrinter: a web based software for the identification of conserved non-coding genome sequence regions between C. elegans and C. briggsae</div><div class=post-meta><span title='2005-12-01 00:00:00 +0000 UTC'>December 1, 2005</span>&nbsp;·&nbsp;Rambaldi Davide, Alessandro Guffanti, Paolo Morandi, Giuseppe Cassata</div></header><figure class=entry-cover><img loading=lazy src=https://tucano.github.io/cover.png alt="NemaFootPrinter: a web based software for the identification of conserved non-coding genome sequence regions between C. elegans and C. briggsae"><p>CBrothersSlider.java is a small application written in Java to interactively display gene structures with intron/exon structure and to select subsequences. The interface display clone identifiers (A) and gene images generated On-the fly (B). Shifting the sliders (C) or submitting directly chromosome coordinates (D), the user is able to select a subsequence. A small control panel (E) can be used to select only the region on the left of the gene (left checkbox) or on the right of the gene (right checkbox); if both the checkboxes are selected, the application selects only the gene sequence. After sequence manipulation, using the Submit button (F), user can post the selected subsequence coordinates to the main script that generates new FASTA files and display the Results page.</p></figure><div class=toc><details><summary accesskey=c title="(Alt + C)"><span class=details>Table of Contents</span></summary><div class=inner><nav id=TableOfContents><ul><li><ul><li><a href=#summary>Summary</a></li></ul></li></ul></nav></div></details></div><div class=post-content><h1 id=abstract>ABSTRACT<a hidden class=anchor aria-hidden=true href=#abstract>#</a></h1><h3 id=summary>Summary<a hidden class=anchor aria-hidden=true href=#summary>#</a></h3><h4 id=background>Background<a hidden class=anchor aria-hidden=true href=#background>#</a></h4><p>NemaFootPrinter (Nematode Transcription Factor Scan Through Philogenetic Footprinting) is a web-based software for interactive identification of conserved, non-exonic DNA segments in the genomes of C. elegans and C. briggsae. It has been implemented according to the following project specifications:a) Automated identification of orthologous gene pairs. b) Interactive selection of the boundaries of the genes to be compared. c) Pairwise sequence comparison with a range of different methods. d) Identification of putative transcription factor binding sites on conserved, non-exonic DNA segments.</p><h4 id=results>Results<a hidden class=anchor aria-hidden=true href=#results>#</a></h4><p>Starting from a C. elegans or C. briggsae gene name or identifier, the software identifies the putative ortholog (if any), based on information derived from public nematode genome annotation databases. The investigator can then retrieve the genome DNA sequences of the two orthologous genes; visualize graphically the genes&rsquo; intron/exon structure and the surrounding DNA regions; select, through an interactive graphical user interface, subsequences of the two gene regions. Using a bioinformatics toolbox (Blast2seq, Dotmatcher, Ssearch and connection to the rVista database) the investigator is able at the end of the procedure to identify and analyze significant sequences similarities, detecting the presence of transcription factor binding sites corresponding to the conserved segments. The software automatically masks exons.</p><h4 id=discussion>Discussion<a hidden class=anchor aria-hidden=true href=#discussion>#</a></h4><p>This software is intended as a practical and intuitive tool for the researchers interested in the identification of non-exonic conserved sequence segments between C. elegans and C. briggsae. These sequences may contain regulatory transcriptional elements since they are conserved between two related, but rapidly evolving genomes. This software also highlights the power of genome annotation databases when they are conceived as an open resource and the possibilities offered by seamless integration of different web services via the http protocol.</p><h4 id=pubmed>PUBMED<a hidden class=anchor aria-hidden=true href=#pubmed>#</a></h4><p><a href=https://pubmed.ncbi.nlm.nih.gov/16351749/>NemaFootPrinter: a web based software for the identification of conserved non-coding genome sequence regions between C. elegans and C. briggsae</a></p><p><strong>BMC Bioinformatics</strong>. 2005 Dec 1;6 Suppl 4(Suppl 4):S22.
doi: 10.1186/1471-2105-6-S4-S22.</p><h4 id=pdf>PDF<a hidden class=anchor aria-hidden=true href=#pdf>#</a></h4><p><a href=nemafootprinter.pdf>NemaFootPrinter.pdf</a></p></div><footer class=post-footer><ul class=post-tags><li><a href=https://tucano.github.io/tags/paper/>Paper</a></li><li><a href=https://tucano.github.io/tags/bioinformatics/>Bioinformatics</a></li></ul><nav class=paginav><a class=prev href=https://tucano.github.io/papers/splicy/><span class=title>« Prev</span><br><span>Splicy</span></a>
<a class=next href=https://tucano.github.io/papers/others/><span class=title>Next »</span><br><span>Other pubblications (collaborations)</span></a></nav></footer></article></main><a href=#top aria-label="go to top" title="Go to Top (Alt + G)" class=top-link id=top-link accesskey=g><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentcolor"><path d="M12 6H0l6-6z"/></svg></a><footer class=footer>:..DAVIDE../TUCANO\..RAMBALDI..: |
<span>&copy; 2022 <a href=https://tucano.github.io/>Tucano Profile Page</a></span>
<span>Powered by
<a href=https://gohugo.io/ rel="noopener noreferrer" target=_blank>Hugo</a> &
        <a href=https://github.com/adityatelange/hugo-PaperMod/ rel=noopener target=_blank>PaperMod</a></span></footer><script>let menu=document.getElementById("menu");menu&&(menu.scrollLeft=localStorage.getItem("menu-scroll-position"),menu.onscroll=function(){localStorage.setItem("menu-scroll-position",menu.scrollLeft)}),document.querySelectorAll('a[href^="#"]').forEach(e=>{e.addEventListener("click",function(e){e.preventDefault();var t=this.getAttribute("href").substr(1);window.matchMedia("(prefers-reduced-motion: reduce)").matches?document.querySelector(`[id='${decodeURIComponent(t)}']`).scrollIntoView():document.querySelector(`[id='${decodeURIComponent(t)}']`).scrollIntoView({behavior:"smooth"}),t==="top"?history.replaceState(null,null," "):history.pushState(null,null,`#${t}`)})})</script><script>var mybutton=document.getElementById("top-link");window.onscroll=function(){document.body.scrollTop>800||document.documentElement.scrollTop>800?(mybutton.style.visibility="visible",mybutton.style.opacity="1"):(mybutton.style.visibility="hidden",mybutton.style.opacity="0")}</script><script>document.getElementById("theme-toggle").addEventListener("click",()=>{document.body.className.includes("dark")?(document.body.classList.remove("dark"),localStorage.setItem("pref-theme","light")):(document.body.classList.add("dark"),localStorage.setItem("pref-theme","dark"))})</script></body></html>